src/detect-ftpbounce.c

```diff
@@ -117,6 +117,7 @@ static int DetectFtpbounceMatchArgs(
     /* Search for the first IP octect(Skips "PORT ") */
     while (i < payload_len && !isdigit((unsigned char)c[i])) i++;
 
+    ip_orig = SCByteSwap32(ip_orig);
```

Comment:
this unconditionally swaps the bytes, won't this break things on little endian? ntohl might be more appropriate

Reply:
`SCNtohl` actually

Reply:
I changed the ip address from host to network order but it didn't look like there was a Suricata specific htonl.

Reply:
@coledishington could you use `SCNtohl` ?

Reply:
@catenacyber Yep, the functions htonl() to ntohl()/SCNtohl() can be used interchangeably although for the sake of readability I choose to use htonl() rather than ntohl()/SCNtohl(), as the parsed FTP IP address is being translated from host-order to network-order.

I can use SCNtohl() in the patch if Suricata prefers SCNtohl() for all byte-order conversions, rather than just network-order to host-order.

Thanks